pkg/occlient/occlient.go

```diff
@@ -2177,11 +2177,19 @@ func getContainerPortsFromStrings(ports []string) ([]corev1.ContainerPort, error
 // envVars is the array containing the key=value pairs
 func getInputEnvVarsFromStrings(envVars []string) ([]corev1.EnvVar, error) {
 	var inputEnvVars []corev1.EnvVar
+	var keys = make(map[string]int)
 	for _, env := range envVars {
 		splits := strings.Split(env, "=")
 		if len(splits) < 2 || len(splits) > 2 {
 			return nil, errors.New("invalid syntax for env, please specify a key=value pair")
 		}
+		_, ok := keys[splits[0]]
+		if ok {
+			return nil, errors.Errorf("multiple values found for variable: %s", splits[0])
```

Comment:
> multiple keys found for variable

I think it's confusing because of the other word "multiple" used before keys. It seems better before "values" WDYT?

> Multiple values found for key: %s", key

The other error messages start with small letters, so I think it's better to keep this small too. WDYT?

Reply:
lets not use word key here.  It is variable name and value. Look at it from the users point of view.

Reply:
Done